test/integration/test/language-id/get-job.test.js

```diff
@@ -0,0 +1,35 @@
+const clientHelper = require('../../src/client-helper');
+const LanguageIdJob = require('../../../dist/src/models/language-id/LanguageIdJob')
+const client = clientHelper.getLanguageIdClient();
+
+beforeAll(async (done) => {
+    const jobList = await client.getListOfJobs();
+    if (jobList === undefined || jobList.length < 2) {
+        const options1 = new Object();
+        options1.metadata = 'Node sdk submit url 1';
+        const options2 = new Object();
+        options2.metadata = 'Node sdk submit url 2';
+        await client.submitJobUrl('https://www.rev.ai/FTC_Sample_1.mp3', options1);
+        await client.submitJobUrl('https://www.rev.ai/FTC_Sample_1.mp3', options2);
+    }
+    done();
+}, 60000);
+
+test('Can get list of jobs', async () => {
+    const jobList = await client.getListOfJobs();
+    jobList.forEach((languageIdJob) => {
```

Comment:
can we loop with an index to check that we see metadata for url 2 before url 1

Reply:
We can't since we submit other jobs for the other language id integration tests, so when we `getListOfJobs` we get all of those ones too.

Reply:
then we should check decreasing createdOn times